coilsnake/modules/eb/SkipNamingModule.py

```diff
@@ -36,6 +36,7 @@ def write_loader_asm(self, rom, offset, s, strlen, mem_offset, byte2):
     def write_to_rom(self, rom):
         if self.data["Enable Skip"]:
             rom[0x1faae] = 0x5c
+            rom[0x1f8f1] = 0x10
```

Comment:
Could you add a comment explaining what this line does? Thanks!

Reply:
```suggestion
            # this fixes the naming screen music playing briefly when skip naming is on
            # it works by changing the jump from @CHANGE_TO_NAMING_SCREEN_MUSIC to @UNKNOWN18 (which normally runs directly after the music change)
            # https://github.com/Herringway/ebsrc/blob/87f514cb4b77fa3193bcb122ea51f5de5cfdd9cf/src/intro/file_select_menu_loop.asm#L101
```

Added, does this look good?